nimrod/test_suite_generation/generators/evosuite_differential_test_suite_generator.py

```diff
@@ -20,9 +18,16 @@ def _execute_tool_for_tests_generation(self, input_jar: str, output_path: str, s
               '-projectCP', input_jar,
               f'-Dregressioncp={scenario.scenario_jars.base}',
               '-class', class_name,
-              f'-Dsearch_budget={self.SEARCH_BUDGET}',
               '-DOUTPUT_DIR=' + output_path,
+              '-Dminimize=false',
+              '-Djunit_check=false',
+              '-Dinline=false',
           ]
+          
+          if use_determinism:
+            params += ["-Dstopping_condition=MaxStatements", f"-seed={self.SEED}"]
+          else:
+            params += [f'-Dsearch_budget={self.SEARCH_BUDGET}']
```

Comment:
Quando for informado a seed, por quanto tempo a ferramenta irá realizar suas funções? 
Eu pensei que tivesse como determinar o tempo da busca (search_budget) mesmo definindo uma seed única.

Reply:
Não, Léuson! Se utilizar o tempo de busca, a execução fica não-determinística, porque o tempo da busca pode variar dependendo da capacidade computacional. Nesse caso então, é necessário utilizar um outro critério de parada, por padrão MaxStatements.

Em outras palavras, trocamos o determinismo do tempo gasto na execução da ferramenta, pelo determinismo da suíte gerada no final :D

Reply:
hum, importante ponto @leusonmario . seria bom explicar isso no readme @jpedroh . a bronca é que 5min na minha maquina pode gerar X testes, e o mesmo tempo na tua maquina @leusonmario pode gerar Y testes

Reply:
Adicionei uma nota com relação a isso na documentação da configuração!